01-introduction.Rmd

```diff
@@ -17,7 +17,7 @@ A major aim of this book is to make geographical data analysis more accessible.
 R is a flexible language that allows access to many spatial software libraries (see section \@ref(why-geocomputation-with-r)).
 Before going into the details of the software, however, it is worth taking a step back and thinking about what we mean by geocomputation.
 
-## What is geocomputation?
+## A definition of geocomputation, and why it is in the book's title
```

Comment:
You are right, the more concise the better! I prolonged the title because the second part of this section mainly deals with possible other titles for the book, and less about geocomputation. But of course, we can stick with `What is geocomputation?`. Here are just a few alternatives:

- `What geocomputation means in this book`
- `This book's definition of geocomputation`
- `Defining geocomputation as a part of Geography`